### PR TITLE
New Phoenix Mix Cheat Sheet

### PR DESCRIPTION
**Pull request by pjhampton**
## Phoenix Mix Cheat Sheet

https://duck.co/ia/view/phoenix_mix_cheat_sheet

### Description
A cheat sheet for searching built-in Phoenix-specific Mix tasks.

### Data Source
http://www.phoenixframework.org/docs/mix-tasks

### Example Queries 
phoenix mix tasks

### Triggers
phoenix mix tasks, phoenix mix commands, mix phoenix.gen

**Screenshot 1 (Dark theme) not-collapsed**
screen shot 2016-02-23 at 21 27 28

**Screenshot 2 (Default theme) collapsed**
screen shot 2016-02-23 at 21 27 40

https://duck.co/ia/view/phoenix_mix_cheat_sheet

### CODE REVIEW

**Comment by daxtheduck** [comment]
@pjhampton Hey!
        
Thanks for taking the time to contribute! We really appreciate it.

We work closely with every contributor to make Instant Answers the best they can be, so we appreciate your patience as we look over your code. From here, the process usually goes like this:

1. Pull Request is reviewed by the DuckDuckGo staff and community
2. Staff and community will leave feedback with any necessary updates to the function or design.
3. Once you've made any necessary corrections, then your Instant Answer will be merged and deployed live on DuckDuckGo!

If you have any questions along the way, feel free to ask them here. Our staff and community are also available on Slack to answer any questions you may have. If you'd like to join us there please send an email to <quackslack@duckduckgo.com>.

>More Info: https://duck.co/duckduckhack/submission_and_review

Thanks!

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by GuiltyDolphin** [comment]
@pjhampton Hey! Thanks for all the cheat sheets you've been doing - we'll be reviewing these over the coming days.

You definitely look like you know your stuff when it comes to FP and AI - the cheat sheets are amazing, but there's more! Would you be interested in doing some full Perl-backed Goodies, or maybe some Spices? I think the community would really love that - and maybe you can get some awesome ideas from your fellow devs.

:dolphin:

**Comment by pjhampton** [comment]
@GuiltyDolphin: Hey Ben. Thanks for the kind words. Hopefully they all go though, but if not let me know what I can do to improve them :smile:

I have an idea for a spice but I want to chat it over with someone like yourself before I submit it to the IA page.

**Comment by GuiltyDolphin** [comment]
@pjhampton Great to hear! I (and others) are available on Slack to chat, so just give me a DM and I'll be happy to talk.

File: share/goodie/cheat_sheets/phoenix_mix.json [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    "id": "phoenix_mix_cheat_sheet",
+    "name": "Phoenix Mix Tasks",
+    "description": "Built-in Phoenix-specific Mix tasks",
+    "metadata": {
+        "sourceName": "Phoenix",
+        "sourceUrl": "http://www.phoenixframework.org/docs/mix-tasks"
+    },
+    "aliases": [
+        "Phoenix Mix Tasks",
+        "Phoenix Mix"
```
Comment (moollaza) [inline share/goodie/cheat_sheets/phoenix_mix.json]: no need for this, the default trigger is `phoenix mix` based on the filename

File: share/goodie/cheat_sheets/phoenix_mix.json [hunk 2]
```diff
@@ -0,0 +1,53 @@
+{
```
Comment (moollaza) [inline share/goodie/cheat_sheets/phoenix_mix.json]: please change the filename to `phoenix-mix.json`

**Comment by moollaza** [comment]
@pjhampton once the filename is fixed this should be good to go!

**Comment by pjhampton** [comment]
Thanks @moollaza - all done :+1:

File: share/goodie/cheat_sheets/json/phoenix-mix.json [hunk 3]
```diff
@@ -0,0 +1,51 @@
+{
+    "id": "phoenix_mix_cheat_sheet",
+    "name": "Phoenix Mix Tasks",
+    "description": "Built-in Phoenix-specific Mix tasks",
+    "metadata": {
+        "sourceName": "Phoenix",
+        "sourceUrl": "http://www.phoenixframework.org/docs/mix-tasks"
+    },
+    "aliases": [
```
Comment (GuiltyDolphin) [inline share/goodie/cheat_sheets/json/phoenix-mix.json]: We can remove this if it is empty.

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=Phoenix Mix Cheat Sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/phoenix_mix_cheat_sheet_mobile.png)

**Comment by moollaza** [comment]
@pjhampton Thanks! This LGTM :+1:

**Comment by daxtheduck** [comment]
@pjhampton this is now merged! It should be live on DuckDuckGo within one week (or during the next release) and we'll be sure to notify you when it happens.

After your Instant Answer goes live for the World to use, you will be able to see [traffic information](http://docs.duckduckhack.com/maintaining/tools.html#monitor-your-instant-answer-page) about your IA as well as receive feedback directly from users. 
                
Please refer to the [maintenance guide and docs](http://docs.duckduckhack.com/maintaining/guidelines.html) to keep up with potential changes to your Instant Answer.
                
Thank you again for your help!

**Pull request merged**